pkg/api/user_token.go

```diff
@@ -132,7 +132,7 @@ func (hs *HTTPServer) revokeUserAuthTokenInternal(c *models.ReqContext, userID i
 		return response.Error(400, "Cannot revoke active user auth token", nil)
 	}
 
-	err = hs.AuthTokenService.RevokeToken(c.Req.Context(), token)
+	err = hs.AuthTokenService.RevokeToken(c.Req.Context(), token, false)
```

Comment:
I think it might make sense to use soft revocation for manual signouts as well, so that the user hit by it gets feedback on what happened. But in that case we would need information about what the reason for the revocation was.

Reply:
@xlson That seems like something that could be introduced in a later PR to keep the scope limited or would you say it's actionable already in this PR?

Reply:
Yeah, I agree with both of you. It may be interesting to start using the soft revocation for other purposes, but as Emil mentioned, I'd prefer to introduce it in a separate PR as a separate issue.

Thanks for the feedback anyway! 😇